Userland/Libraries/LibJS/Tests/builtins/ArrayBuffer/ArrayBuffer.prototype.slice.js

```diff
@@ -0,0 +1,29 @@
+test("single parameter", () => {
+    const buffer = new ArrayBuffer(16);
+    const fullView = new Int32Array(buffer);
+
+    // modify some value that we can check in the sliced buffer
+    fullView[3] = 7;
+
+    // slice the buffer and use a new int32 view to perform basic checks
+    const slicedBuffer = buffer.slice(12);
+    const slicedView = new Int32Array(slicedBuffer);
+
+    expect(slicedView.length).toBe(1);
```

Comment:
Please use this instead:

```suggestion
    expect(slicedView).toHaveLength(1);
```